deploy/packages/parallel-libraries.yaml

```diff
@@ -87,3 +87,4 @@ packages:
       - python
     specs:
       - neuron@7.6.6
+      - neuron+debug@7.6.6
```

Comment:
`deploy/configs/applications/modules.yaml` should be updated to generate module for non-debug version only i.e. whitelist `neuron~debug@7.6.6`

Reply:
Modules are generated by passing the spec list generated from this `packages` configuration. We should not put versions into `modules.yaml`.

Reply:
oh yes...realise this yesterday after discussing with Ioannis.

@matz-e  : if spec has `neuron` (with implicit `~debug`) and `neuron+debug` and modules.yaml has `neuron`, I assume result will be `neuron~debug`?